Templates/make_jenkins_EC2-Agent-Windows-autoscale.tmplt.json

```diff
@@ -0,0 +1,1620 @@
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Instance Configuration"
+                    },
+                    "Parameters": [
+                        "AmiId",
+                        "AppVolumeMountPath",
+                        "AppVolumeSize",
+                        "AppVolumeType",
+                        "InstanceType",
+                        "InstanceRole",
+                        "KeyPairName",
+                        "NoPublicIp",
+                        "NoReboot",
+                        "NoUpdates",
+                        "PipIndexFips",
+                        "SecurityGroupIds",
+                        "SubnetIds"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Watchmaker Configuration"
+                    },
+                    "Parameters": [
+                        "PythonInstaller",
+                        "PypiIndexUrl",
+                        "WatchmakerBootstrapper",
+                        "WatchmakerConfig",
+                        "WatchmakerEnvironment",
+                        "WatchmakerOuPath",
+                        "WatchmakerComputerName",
+                        "WatchmakerAdminGroups",
+                        "WatchmakerAdminUsers"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Application Configuration"
+                    },
+                    "Parameters": [
+                        "AppScriptUrl",
+                        "AppScriptParams"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "EC2 Application EBS Volume"
+                    },
+                    "Parameters": [
+                        "AppVolumeDevice",
+                        "AppVolumeSize",
+                        "AppVolumeType"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "Network Configuration"
+                    },
+                    "Parameters": [
+                        "PrivateIp",
+                        "SubnetIds"
+                    ]
+                },
+                {
+                    "Label": {
+                        "default": "CloudFormation Configuration"
+                    },
+                    "Parameters": [
+                        "CfnEndpointUrl",
+                        "CloudWatchAgentUrl",
+                        "ToggleCfnInitUpdate"
+                    ]
+                }
+            ],
+            "ParameterLabels": {
+                "ToggleCfnInitUpdate": {
+                    "default": "Force Cfn Init Update"
+                }
+            }
+        },
+        "Version": "1.5.2"
+    },
+    "Outputs": {
```

Comment:
Seems like the `Outputs` section is still based on the `Instance` template.  Take a look at the Gitlab autoscale template as a reference for what is being used.

https://github.com/plus3it/cfn-gitlab/blob/1b02d797c0ae6251d4d004a7e6a7597a0c3b5b13/Templates/make_gitlab_EC2-autoscale.tmplt.json#L291

You can also take a look at the Watchmaker template for a Windows reference:

https://github.com/plus3it/terraform-aws-watchmaker/blob/e5e5b28641907163de203f1039eb9cc2baae3768/modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml#L143

The latest Watchmaker templates have been converted to YAML but it'll give you an idea of the outputs being used.